tests/hollow_sphere.prm

```diff
@@ -0,0 +1,99 @@
+# The 3D HollowSphere benchmark, for which an 
+# analytical solution is available (Thieulot, in prep). 
+
+
+############### Global parameters
+
+set Additional shared libraries            = ./libhollow_sphere.so
+set Linear solver tolerance                = 1e-12
+set Dimension                              = 3
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = false
+set Nonlinear solver scheme                = Stokes only
+set Output directory                       = output
+set Pressure normalization                 = surface
+set Number of cheap Stokes solver steps    = 0
+
+############### Parameters describing the model
+# The setup is a 3D shell 
+# Because the temperature plays no role in this model we need not
+# bother to describe temperature boundary conditions or
+# the material parameters that pertain to the temperature.
+
+subsection Geometry model
+  set Model name = spherical shell
+
+  subsection Spherical shell
+    set Inner radius = 0.5
+    set Outer radius = 1.0
+    set Cells along circumference = 12
+  end
+
+end
+
+#Boundary conditions
+subsection Model settings
+
+  set Prescribed velocity boundary indicators = inner : HollowSphereBoundary, \
+                                                outer : HollowSphereBoundary
+end
+
+
+subsection Material model
+  set Model name = HollowSphereMaterial
+end
+
+
+subsection Gravity model
+  set Model name = HollowSphereGravity
+end
+
+subsection HollowSphere benchmark
+   #Viscosity parameter is m 
+   set Viscosity parameter             = -1
+end 
+
+############### Parameters describing the temperature field
+# As above, there is no need to set anything for the
+# temperature boundary conditions.
+
+subsection Boundary temperature model
+  set Model name = box 
+end
+
+subsection Initial temperature model 
+  set Model name = function
+
+  subsection Function
+    set Function expression = 0
+  end
+end
+
+
+############### Parameters describing the discretization
+# The following parameters describe how often we want to refine
+# the mesh globally and adaptively, what fraction of cells should
+# be refined in each adaptive refinement step, and what refinement
+# indicator to use when refining the mesh adaptively. 
+
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 2
+  set Refinement fraction                = 0.2
+  set Strategy                           = velocity
+end
+
+
+############### Parameters describing what to do with the solution
+# The final section allows us to choose which postprocessors to
+# run at the end of each time step. 
+
+subsection Postprocess
+  set List of postprocessors = visualization, velocity statistics, HollowSpherePostprocessor, depth average, memory statistics
```

Comment:
Can you remove the memory postprocessor and update the results for both tests, please? The number is unlikely to be very stable.